### PR TITLE
Fix help message in vmprofshow for prune options

### PR DESCRIPTION
**Pull request by pankajp**

### CODE REVIEW

File: vmprof/show.py [hunk 1]
```diff
@@ -232,13 +232,13 @@ def main():
         '--prune_percent',
         type=float,
         default=0,
-        help="The indention per level within the call graph.")
+        help="Prune output of a profile stats node below specified CPU smaples.")
```
Comment (rlamy) [inline vmprof/show.py]:
```suggestion
        help="Prune output of a profile stats node below specified CPU samples.")
```

**Comment by rlamy** [comment]
LGTM, apart from the one typo.
Sorry for the late review.

**Comment by pankajp** [comment]
@rlamy Thanks for the review. I've fixed the typo

**Comment by rlamy** [comment]
Thank you!

**Pull request merged**